DataStructures.Tests/AATreeTests.cs

```diff
@@ -212,7 +212,7 @@ public void GetKeysPostOrder_NonEmptyTree_ReturnsCorrectAnswer()
 
             var expected = new [] { 1, 3, 2, 5, 7, 10, 9, 8, 6, 4 };
             var actual = tree.GetKeysPostOrder();
-            Assert.IsTrue(Enumerable.SequenceEqual(expected, actual));
+            Assert.IsTrue(expected.SequenceEqual(actual));
```

Comment:
Could you use FluentAssertions? They are much more eloquent

Reply:
@siriak tests for AAtree are rewritten using fluent assertions